gulpfile.js

```diff
@@ -9,15 +9,14 @@
 
 var gulp = require('gulp');
 
-var buildModules = __dirname + '/node_modules/fxos-build/node_modules/';
```

Comment:
That works for me. `fxos-build` has been super helpful at the beginning to start with a working development workflow. But now we need something more custom and as we grow out of the prototype phase we can dedicate more time to a solid build system.